### ISSUE TITLE
Trying to attach a TransitionEffect without name or group specified

### ISSUE THREAD

**Issue by toverux** [GiampaoloGabba/Xamarin.Plugin.SharedTransitions#32]
## Bug Information

Version Number of Plugin: 2.1.0
Device Tested On: Android
Version of Xamarin: 4.5.0.617

### Steps to reproduce the Behavior

I attached the effect through Transition.Name on an element:

```xaml
<views:CoverHeader sharedTransitions:Transition.Name="RoomPicture" />
```

In the second page, the exact same thing.

Pages are pushed onto a navigation page that derives from `SharedTransitionNavigationPage`.

### Expected Behavior

Makes the transition.

### Actual Behavior

No transition is made and there's this in the logs:

```
===== SAHARED: update property for Aterno.Views.CoverHeader:  - RoomPicture
... a bit later ...
Trying to attach a TransitionEffect without name or group specified. Nothing done
```

**Comment by GiampaoloGabba** [comment]
sorry for the delay :)
This debug information was not in the right place and was behing executed everytime:
`Trying to attach a TransitionEffect without name or group specified. Nothing done`

So ignore it (this is gone in the next release) :)

Is your element inside a Collection of items?  What's inside the coverheader?
It should work, maybe try to apply the effect in a base element (not a custom view) and see what happens.

**Comment by toverux** [comment]
Hi @GiampaoloGabba, no probem, thanks.
I tried again and this time, found the obvious mistake I made the last time.

Like I said, I *inherited* `SharedTransitionNavigationPage`, I did not use it directly.
Obviously, I had a custom renderer attached to my own custom navigation page, and that one was still inheriting from `NavigationPageRenderer` ; the only thing I missed the last time was replacing this renderer with `SharedTransitionNavigationPageRenderer`.

Very sorry to have bothered you with that!
